SignalServiceKit/tests/Util/SSKSwiftTests.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+import XCTest
+
+class SSKSwiftTests: XCTestCase {
```

Comment:
Why is there a separate swift base class? And don't we want it to do the same things as the ObjC base class re: environment setup/teardown?

Reply:
> Why is there a separate swift base class?

Sorry, I should have left a comment.  This was just to workaround to get things building.  It's been 2+ weeks since I did this and I can't recall the details, but it's something like this: if you don't have any .swift tests, no Swift-to-ObjC bridging doesn't work.  Once I get things building again, I'll improve the comment.